contexts/NetworksContext.tsx

```diff
@@ -17,13 +36,79 @@ export const NetworkContext = createContext<INetworksContext>({
   setNetwork: () => null,
 });
 
-const NetworksProvider = ({ children }: { children: React.ReactNode }) => {
-  const [network, setNetwork] = useState<INetwork>({
-    // Default Network
-    name: NetworkName.MAINNET,
-    rpc: RPC_URL[NetworkName.MAINNET],
-    imgUrl: '/lukso.png',
-  });
+const NetworksProvider = ({ children }) => {
+  const router = useRouter();
+
+  const getNetworkFromLocalStorage = (): INetwork => {
+    if (typeof window !== 'undefined') {
+      const storedNetworkName = localStorage.getItem('erc725InspectNetwork');
```

Comment:
Not sure to understand. Why is this saved in the local storage?
`useContext` should enable to save that in a context variable and re-import it in other pages